_data/cryptocurrencies.yml

```diff
@@ -44,6 +53,14 @@ websites:
       hardware: Yes
       doc: https://bitgo.zendesk.com/hc/en-us/sections/201009049-Two-Factor-Authentication-Help-Page
 
+    - name: Bithumb
+      url: https://www.bithumb.com/
+      img: bithumb.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: http://bithumb.cafe/archives/7286
```

Comment:
Seems that they support HTTPS (even though some stuff still load HTTP). Could you change the doc to `https://bithumb.cafe/archives/7286 `?